networking/network_observability/network-observability-overview.adoc

```diff
@@ -0,0 +1,50 @@
+:_content-type: ASSEMBLY
+[id='network-observability-overview']
+= About Network Observability
+include::_attributes/common-attributes.adoc[]
+:context: network-observability-overview
+
+toc::[]
+
+Red Hat offers cluster administrators the Network Observability Operator to observe the network traffic for {product-title} clusters. The Network Observability uses the eBPF technology to create network flows. The network flows are then enriched with {product-title} information and stored in Loki. You can view and analyze the stored netflow information in the {product-title} console for further insight and troubleshooting.
+
+[id="dependency-network-observability"]
+== Dependency of Network Observability Operator
+
+The Network Observability Operator requires the following Operators:
+
+* Loki: You must install Loki. Loki is the backend that is used to store all collected flows. It is recommended to install the Loki Operator for the installation of Network Observability Operator.
```

Comment:
2 conflicting statements here. I think you can remove `It is recommended to install the Loki Operator for the installation of Network Observability Operator.` Instead write `You must install the Loki Operator for the installation of Network Observability Operator.` in the beginning of the sentence.

Reply:
@snarayan-redhat 
Actually Loki the backend is mandatory but Loki Operator is not a must-have. That's why Developers suggested writing "It is recommended"

Reply:
@xenolinux I tend to agree here. It is not clear that we recommend installing the Loki Operator to install Loki. Now, it seems like two steps: Install Loki; Install Loki Operator. Would this make sense?  
_It is recommended to install Loki by installing the Loki Operator._

Reply:
Addressed